themes/tdr/login/login.ftl

```diff
@@ -37,8 +36,11 @@
 
 
             <button class="govuk-button" type="submit" data-module="govuk-button" role="button" name="login">
-                ${msg("continueButton")}
+                ${msg("signInButton")}
             </button>
+              <p class="govuk-body-s">
+                  <a href="mailto:tdr@nationalarchives.gov.uk" data-hsupport="email" class="govuk-link">I need to reset my password</a>
```

Comment:
Linda mentioned that in the design meeting yesterday you decided to always show the email address when you're adding a mailto link. Is that right, or did I misunderstand what she meant?

Reply:
I'm still chasing this up.

Reply:
Any updates on this?

Reply:
Going to ask Kirsten in our catch up now because Linda wasn't sure if she has the latest update on this.

Reply:
We decided to link to the contact page instead.